src/constants.rs

```diff
@@ -509,30 +509,20 @@ bitflags! {
         const TRIGGERPREIN = 1 << 25;  // %triggerprein dependency
         const KEYRING	= 1 << 26;
         // bit 27 unused
-        const CONFIG	= 1 << 28;
+        const CONFIG	= 1 << 28;    // config() dependency
         const META	= 1 << 29;	      // meta dependency
     }
 }
 
 bitflags! {
     /// Flags to configure scriptlet execution,
-    ///
     #[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
     pub struct ScriptletFlags: u32 {
-        /// Macro expansion,
-        ///
-        /// Corresponds to RPMSCRIPT_FLAG_EXPAND
-        ///
+        /// Macro expansion
         const EXPAND = 1;
-        /// Header queryformat expansion,
-        ///
-        /// Corresponds to RPMSCRIPT_FLAG_QFORMAT
-        ///
+        /// Header queryformat expansion
         const QFORMAT = 1 << 1;
-        /// Critical for success/failure,
-        ///
-        /// Corresponds to RPMSCRIPT_FLAG_CRITICAL
```

Comment:
Since there is no rpm spec, I find it helpful to have references to the C code names.